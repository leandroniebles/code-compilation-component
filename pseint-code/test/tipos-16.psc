funcion foo(n)
  n<-5;
finfuncion
proceso tipos
  leer n;
  foo(n);
finproceso