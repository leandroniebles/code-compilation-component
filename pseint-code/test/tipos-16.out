> "hola"
=== Line 2: ExeError 125
